src/main/java/com/hazelcast/cloud/JdbcClient.java

```diff
@@ -0,0 +1,66 @@
+package com.hazelcast.cloud;
+
+import com.hazelcast.client.HazelcastClient;
+import com.hazelcast.client.config.ClientConfig;
+import com.hazelcast.cloud.model.City;
+import com.hazelcast.config.SSLConfig;
+import com.hazelcast.core.HazelcastInstance;
+
+import java.net.URISyntaxException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.Properties;
+
+import static com.hazelcast.cloud.ClientWithSsl.*;
+
+public class JdbcClient {
+
+    public static void main(String[] args) throws URISyntaxException {
+        // Configure the client
+        ClassLoader classLoader = JdbcClient.class.getClassLoader();
+        Properties tlsProperties = new Properties();
+        tlsProperties.setProperty("javax.net.ssl.keyStore", classLoader.getResource("client.keystore").toURI().getPath());
+        tlsProperties.setProperty("javax.net.ssl.keyStorePassword", "YOUR_SSL_PASSWORD");
+        tlsProperties.setProperty("javax.net.ssl.trustStore",
+                classLoader.getResource("client.truststore").toURI().getPath());
+        tlsProperties.setProperty("javax.net.ssl.trustStorePassword", "YOUR_SSL_PASSWORD");
+
+        ClientConfig config = new ClientConfig();
+        config.getNetworkConfig().setSSLConfig(new SSLConfig().setEnabled(true).setProperties(tlsProperties));
+        config.getNetworkConfig().getCloudConfig()
+                .setDiscoveryToken("YOUR_CLUSTER_DISCOVERY_TOKEN")
+                .setEnabled(true);
+        config.setProperty("hazelcast.client.cloud.url", "YOUR_DISCOVERY_URL");
+        config.setClusterName("YOUR_CLUSTER_NAME");
+
+        HazelcastInstance client = HazelcastClient.newHazelcastClient(config);
```

Comment:
I know we did it for code-reuse but it feels weird to start a separate client instance for the JDBC code sample. What do you think about it? Should we just use the JDBC client in this code sample?

Reply:
@mdumandag I principally agree with you, but officially JDBC client is more limited than the normal one -- as it is in Beta stage. For instance, officially it is not able to create mappings (although it unofficially does)
https://docs.hazelcast.com/hazelcast/5.2/sql/mapping-to-jdbc
I prefer to stick to the official documentation here.

Without existing mappings and data, it is not as streamlined to just have a select query that will fail on execution due to absence of mappings.

What do you think?

Reply:
OK, that makes sense.